README.md

```diff
@@ -139,6 +139,19 @@ Add the following to `esy.json`:
 }
 ```
 
+#### Neovim >= 0.8
+
+```lua
```

Comment:
This is doing too much. The configuration shouldn't be more than a few lines:

```lua
require('lspconfig')['ocamllsp'].setup {
  cmd = { 'ocamllsp', '--fallback-read-dot-merlin' },
}
```


Setting up the environment is a user problem, e.g. start `nvim` with `esy nvim`, `nix develop #. -c nvim`, `opam exec -- nvim`, etc.